pandas/core/reshape/concat.py

```diff
@@ -94,6 +100,14 @@ def concat(
 
         .. versionadded:: 0.23.0
 
+    suffixes : tuple of str, default None
+        Suffix to apply to overlapping column names for each concatenated object
+        respectively. If the length of suffixes does not match with number of
+        concatenated objects, an error will raise. If None, the output will remain
+        as is with duplicated column names.
```

Comment:
Do we want to replicate the behavior of merge in regards to `suffixes=(False, False)`?

https://github.com/pandas-dev/pandas/blob/e246c3b05924ac1fe083565a765ce847fcad3d91/pandas/core/frame.py#L203-L206